.github/workflows/build-test.yml

```diff
@@ -24,7 +24,8 @@ jobs:
           # renovate: datasource=github-tags depName=defenseunicorns/zarf versioning=semver
           version: v0.32.4
 
-      - run: "curl -s https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash"
+      # renovate: datasource=github-tags depName=k3d-io/k3d versioning=semver
```

Comment:
In [this PR](https://github.com/defenseunicorns/uds-k3d/pull/47/files), I updated this to use the common task for setup, I _think_ it feels acceptable to just track the common task version as a proxy for which k3d version we are using.

Reply:
@rjferguson21 - i like that approach!
perhaps then it makes more sense for me to make similar changes in the common repo and in this README we link to common for which versions to use for k3d and zarf.  what do you think?

Reply:
I think that sounds good.